projects/storefrontapp-e2e-cypress/cypress/support/general.commands.ts

```diff
@@ -44,3 +61,127 @@ Cypress.Commands.add(
       .click();
   }
 );
+
+export interface LoginUserData {
+  firstName?: string;
+  lastName?: string;
+  alias: string;
+  password?: string;
+  titleCode?: string;
+  username?: string;
+}
+
+export interface LoginOptions {
+  alwaysNew?: boolean;
```

Comment:
As discussed offile, `forceRegister` will be more descriptive, as we enforce register only for this call (not for every other calls)